receiver/chronyreceiver/documentation.md

```diff
@@ -0,0 +1,32 @@
+[comment]: <> (Code generated by mdatagen. DO NOT EDIT.)
+
+# chrony receiver
+
+## Metrics
+
+These are the metrics available for this scraper.
+
+| Name | Description | Unit | Type | Attributes |
+| ---- | ----------- | ---- | ---- | ---------- |
+| ntp.frequency.offset | The frequency is the rate by which the system s clock would be wrong if chronyd was not correcting it. It is expressed in ppm (parts per million). For example, a value of 1 ppm would mean that when the system\’s clock thinks it has advanced 1 second, it has actually advanced by 1.000001 seconds relative to true time. | {ppm} | Gauge(Double) | <ul> <li>leap.status</li> </ul> |
```

Comment:
Nit: The quotation isn't escaped properly and it shows up in the rendered markdown. Could be fixed by putting quotes around the whole yaml string.